common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -432,5 +524,61 @@ color_lists = {
 			125 = "yellow"
 			250 = "red_light"
 		}
+		###### Warcraft Culture Specific CoAs ######
+		
+		### Humans ###
+		special_selection = {
+			trigger = { azerothian_coa_trigger = yes }
+			30  = "white"
+			150 = "yellow"
+		}
+		special_selection = {
+			trigger = { lordaeronian_coa_trigger = yes }
+			65  = "yellow"
+			150 = "white"
+		}	
+		special_selection = {
+			trigger = { arathorian_coa_trigger = yes }
+			50  = "yellow"
+			150 = "white"
+		}
+		special_selection = {
+			trigger = { alteraci_coa_trigger = yes }
+			20  = "yellow"
+			50  = "white"
+			50  = "yellow_alterac"
+		}
```

Comment:
I would put more work into Alterac. I couldn't find any CoA that follows their orange + black pattern.
Also, I don't get why there's white.

Reply:
> I couldn't find any CoA that follows their orange + black pattern.
This might have seemed the case for some other cultures as well. I think so myself but I suspect the main cause is because the only colours in `metal colours` are white, yellow and orange_ashvane. As a result, it is impossible to find a black and orange flag. A bright orange or brown to put in `metal colours` does not fit the dark colours of Alterac and I spent a lot of time tuning the colours because CK3 makes all colours brighter in game somehow; when I put brighter colours in the lists, they almost always looked neon and stood out unusually from the other colours in a jarring way.

Reply:
> This might have seemed the case for some other cultures too. I think so myself but suspect the main cause is because the only colours in metal colours are white, yellow and orange_ashvane. As a result, it is impossible to find a black and orange flag. A bright orange or brown to put in metal colours does not fit the dark colours of Alterac and I spent a lot of time tuning the colours because CK3 makes all colours brighter in game somehow; when I put brighter colours in the lists, they almost always looked neon and stood out unusually from the other colours in a jarring way.

Show me how you implemented it. Push on this branch or create a new branch.

> As for white, it is there to provide some variety and represents the influence of nearby Lordaeron.

Why? They're antagonists after all.

Reply:
White removed as a part of a broader thematic consolidation.